internal/server/filesystem/server.go

```diff
@@ -67,6 +67,10 @@ func isAbsWindows(path string) bool {
 	return absPathRegexWindows.MatchString(path)
 }
 
+func (s *Server) ValidateSMBLinkPath(path string) error {
```

Comment:
can we just use validataPathWindows directly?

Reply:
I tried but could not workout the export due to the need to use the Using ValidatePathWindows requires us to export the type - internal.PLUGIN. Due to restrictions around the way internal can be used in go, this was not working when used from smb - would not compile. Hence had to opt to use in an interface instead of using the common code directly.

Reply:
validatePluginPath as discussed?